docs/Gruntfile.js

```diff
@@ -43,17 +43,19 @@ module.exports = function(grunt) {
 			'<script src="/dist/assets/dox.js"></script>',
 			'<script src="/dist/assets/mark.min.js"></script>',
 			'<script src="/dist/assets/jquery-2.2.4.min.js"></script>',
+			'<link rel="stylesheet" href="//127.0.0.1:10111/dist/ts.css"/>',
```

Comment:
can this be a relative `href` so we can test on mobile locally?

Reply:
No, because the file is served on another port and so it is not relative to the website, it's more like a completely different website. That's why this IP address will be replaced with your real, public IP (for example `172.17.0.96:10111`) so that the mobile phone can see the stylesheet even though it is served by another process 👍